test/controllers/insights_cloud/api/cloud_request_controller_test.rb

```diff
@@ -50,9 +50,11 @@ class CloudRequestControllerTest < ActionController::TestCase
 
       mock_composer = mock('composer')
       ::JobInvocationComposer.expects(:for_feature).with do |feature, host_ids, params|
+        actual_ids = host_ids.sort
+        expected_ids = [host1.id, host2.id].sort
         feature == :rh_cloud_connector_run_playbook &&
-        host_ids.first == host1.id &&
-        host_ids.last == host2.id
+        actual_ids.first == expected_ids.first &&
+        actual_ids.last == expected_ids.last
```

Comment:
Would it be better to match the whole array instead?
```suggestion
        expected_ids == actual_ids
```

You can then also optimize it to avoid the additional variables
```suggestion
        [host1.id, host2.id].sort == host_ids.sort
```

Reply:
In Katello I added a helper that does that, `assert_equal_arrays`. guess we can't use it here though.

Reply:
I would prefer to avoid cross-referencing plugins.

Reply:
> In Katello I added a helper that does that, `assert_equal_arrays`. guess we can't use it here though.

This actually isn't an assertion. I made the same mistake first.